src/api/crunchyroll-beta-http-client/crunchyroll-beta-http-client.types.ts

```diff
@@ -0,0 +1,222 @@
+export type ServerSideDataFetching = {
+  enabled: boolean;
+  onEnterExecutionTimeout: number;
+};
+
+export type Logger = {
+  enabled: boolean;
+};
+
+export type Debug = {
+  enabled: boolean;
+};
+
+export type Options = {
+  accountId: string;
+  trustKey: string;
+  agentId: string;
+  licenseKey: string;
+  applicationId: string;
+};
+
+export type BrowserMonitoring = {
+  enabled: boolean;
+  options: Options;
+};
+
+export type Newrelic = {
+  enabled: boolean;
+  browserMonitoring: BrowserMonitoring;
+};
+
+export type Drm = {
+  dashRolloutPercent: number;
+  fairplayCertificateUrl: string;
+  playreadyCertificateUrl: string;
+  widevineCertificateUrl: string;
+};
+
+export type Features = {
+  adConfigId: string;
+  oneAdPerMidroll: string;
+  strictMode: string;
+  vilosUrl: string;
+  bypassOptInFlow: string;
+  defaultLanguage: string;
+  ssr: string;
+  countryOverride: string;
+  resetPassword: string;
+  videoQualitySelector: string;
+  overrideFeedId: string;
+  overrideCarouselId: string;
+  betaNotificationsPage: string;
+  share: string;
+  watchUpsell: string;
+  evidon: string;
+  clientApiDomain: string;
+  watchAllEpisodes: string;
+  accountStateEnforcement: string;
+};
+
+export type DarkFeatureConfig = {
+  features: Features;
+  storageKey: string;
+  securedRoutes: any[];
+};
+
+export type CxApiParams = {
+  apiDomain: string;
+  accountAuthClientId: string;
+  anonClientId: string;
+};
+
+export type RedirectRoute = {
+  url: string;
+  patterns: any[];
+  statusCode: number;
+};
+
+export type PayPal = {
+  env: string;
+};
+
+export type AppConfig = {
+  reCaptchaKey: string;
+  serverSideDataFetching: ServerSideDataFetching;
+  logger: Logger;
+  debug: Debug;
+  newrelic: Newrelic;
+  drm: Drm;
+  darkFeatureConfig: DarkFeatureConfig;
+  crunchyrollSiteUrl: string;
+  staticDomain: string;
+  assetsPath: string;
+  assetsBuildPath: string;
+  baseSiteUrl: string;
+  baseStaticUrl: string;
+  baseSsoUrl: string;
+  vilosPlayerUrl: string;
+  cancelHappyMealUrl: string;
```

Comment:
:?????

Reply:
This was machine translated types so I actually didn't read it lol but since when did Crunchyroll offer happy meals 👀👀